.github/workflows/pull-request.yml

```
@@ -253,6 +253,35 @@
+        
+             - name: Set up Docker
+             run: |
+                  sudo apt-get update
+                  sudo apt-get install -y docker.io
+        
+              - name: Run Docker Container
+                run: |
+                  docker run -d --name talawa-admin-container -p 8080:8080 talawa-admin-app
+        
+              - name: Check if Talawa Admin App is running
+               run: |
```

yamllint

[warning] wrong indentation: expected 16 but found 15

(indentation)